doc/arkode/guide/source/Usage/SPRKStep_c_interface/User_callable.rst

```diff
@@ -0,0 +1,1229 @@
+      * *ARK_ROOT_RETURN* if :c:func:`SPRKStepEvolve()` succeeded, and
+        found one or more roots.  If the number of root functions,
+        *nrtfn*, is greater than 1, call
+        :c:func:`SPRKStepGetRootInfo()` to see which :math:`g_i` were
+        found to have a root at (*\*tret*).
+      * *ARK_TSTOP_RETURN* if :c:func:`SPRKStepEvolve()` succeeded and
+        returned at *tstop*.
+      * *ARK_MEM_NULL* if the *arkode_mem* argument was ``NULL``.
+      * *ARK_NO_MALLOC* if *arkode_mem* was not allocated.
+      * *ARK_ILL_INPUT* if one of the inputs to
+        :c:func:`SPRKStepEvolve()` is illegal, or some other input to
+        the solver was either illegal or missing.  Details will be
+        provided in the error message.  Typical causes of this failure:
+
+        (a) A component of the error weight vector became zero during
+            internal time-stepping.
+
+        (b) A root of one of the root functions was found both at a
+            point :math:`t` and also very near :math:`t`.
+
+        (c) The initial condition violates the inequality constraints.
+
+      * *ARK_TOO_MUCH_WORK* if the solver took *mxstep* internal steps
+        but could not reach *tout*.  The default value for *mxstep* is
+        *MXSTEP_DEFAULT = 500*.
+      * *ARK_TOO_MUCH_ACC* if the solver could not satisfy the accuracy
+        demanded by the user for some internal step.
+      * *ARK_ERR_FAILURE* if error test failures occurred either too many
+        times (*ark_maxnef*) during one internal time step or occurred
+        with :math:`|h| = h_{min}`.
+      * *ARK_VECTOROP_ERR* a vector operation error occurred.
+
+   **Notes:**
+      The input vector *yout* can use the same memory as the
+      vector *y0* of initial conditions that was passed to
+      :c:func:`SPRKStepCreate`.
+
+      In *ARK_ONE_STEP* mode, *tout* is used only on the first call, and
+      only to get the direction and a rough scale of the independent
+      variable. All failure return values are negative and so testing the
+      return argument for negative values will trap all
+      :c:func:`SPRKStepEvolve()` failures.
+
+      Since interpolation may reduce the accuracy in the reported
+      solution, if full method accuracy is desired the user should issue
+      a call to :c:func:`SPRKStepSetStopTime()` before the call to
+      :c:func:`SPRKStepEvolve()` to specify a fixed stop time to
+      end the time step and return to the user.  Upon return from
+      :c:func:`SPRKStepEvolve()`, a copy of the internal solution
+      :math:`y_{n}` will be returned in the vector *yout*.  Once the
+      integrator returns at a *tstop* time, any future testing for
+      *tstop* is disabled (and can be re-enabled only though a new call
+      to :c:func:`SPRKStepSetStopTime()`). SPRKStep uses the ARKODE
+      Lagrange interpolation module by default because testing showed that
+      it does a better job of maintaining conservation than Hermite interpolation.
```

Comment:
> but still introduces additional error compared to the full accuracy solution
OR introduces no additional error.

Reply:
Somewhere later on in the documentation there was a phrasing I thought was clearer. I don't remember where but I did add a comment that I thought it could address this one (and other similar comments).